<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depositcalc</class>
 <widget class="QDialog" name="depositcalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма вклада</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок (Мес.)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка (%)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Нологовая ставка (%)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Периодичность выплат</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deposit_sum">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="period">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="interest_rate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tax_rate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="capitalization">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Капитализация</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма пополнения (&gt;0) / снятия (&lt;0) </string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_or_rm_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>сумма налога</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>сумма на вкладе к концу срока</string>
   </property>
  </widget>
  <widget class="QLabel" name="interest_charges">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="tax_sum">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>50</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="end_sum">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>90</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отчистить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="periodicity">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в 2 месяца</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в 4 месяца</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в полгода</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в год</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="add_or_rm_sum">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер месяца</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>571</width>
     <height>211</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <column>
    <property name="text">
     <string>Номер месяца</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Пополнение / Cнятие</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Номер дня в месяце</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="month_number">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="day_number">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер дня в периоде выплат</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
